### PR TITLE
NISP-2260: Changed wordings for personal max scenario

### PR DESCRIPTION
**Pull request by nehakannaujia**
NISP-2260: Changed wordings for personal max scenario.
@dspork : Please review.

### CODE REVIEW

**Comment by dspork** [comment]
Is it possible to support both versions using the `futureProofPersonalMax` flag in the `ApplicationConfig`?

**Comment by nehakannaujia** [comment]
@dspork  : Changes done please review.

File: app/uk/gov/hmrc/nisp/views/includes/fillGapsFutureProof.scala.html [hunk 1]
```diff
@@ -24,8 +24,7 @@
 
 <h2 class="heading-medium">@Messages("nisp.main.context.fillGaps.improve.title")</h2>
 
-<p>@Messages("nisp.main.context.fillgaps.futureproof.para")</p>
-
-@spChart(Html(Messages("nisp.main.context.fillgaps.futureproof.chart", statePension.amounts.maximum.gapsToFill)), personalMaximumChart)
```
Comment (dspork) [inline app/uk/gov/hmrc/nisp/views/includes/fillGapsFutureProof.scala.html]: Can you remove this message? `nisp.main.context.fillgaps.futureproof.chart`

Comment (nehakannaujia) [inline app/uk/gov/hmrc/nisp/views/includes/fillGapsFutureProof.scala.html]: @dspork  yeah, i missed this.

**Comment by nehakannaujia** [comment]
@dspork  : Added review changes.

**Pull request merged**